compiler/parser.nim

```diff
@@ -170,13 +170,15 @@ proc validInd(p: var Parser): bool {.inline.} =
 proc rawSkipComment(p: var Parser, node: PNode) =
   if p.tok.tokType == tkComment:
     if node != nil:
+      var rhs = node.comment
       when defined(nimpretty):
         if p.tok.commentOffsetB > p.tok.commentOffsetA:
-          node.comment.add fileSection(p.lex.config, p.lex.fileIdx, p.tok.commentOffsetA, p.tok.commentOffsetB)
+          rhs.add fileSection(p.lex.config, p.lex.fileIdx, p.tok.commentOffsetA, p.tok.commentOffsetB)
         else:
-          node.comment.add p.tok.literal
+          rhs.add p.tok.literal
       else:
-        node.comment.add  p.tok.literal
+        rhs.add p.tok.literal
+      node.comment = rhs
```

Comment:
For preOrc Nim this doesn't move and should be ``node.comment = move rhs``. But I don't know why you didn't keep the old version. Impossible with the new accessors?

Reply:
> But I don't know why you didn't keep the old version. Impossible with the new accessors?

indeed, accessors can't work with add, it'd need a custom `add` API, which was only used here